### ISSUE TITLE
List of Construals: language select

### ISSUE THREAD

**Issue by dhwang90** [nert-gu/Xposition#149]
Is it possible to create a function so that one can select the language you want to view for the list of construals page?

**Comment by nschneid** [comment]
Good idea: e.g. /construals/en could show just the construals that apply to English. I don't think this would be hard to implement.
